package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bedrock",
+  "version": "0.1.0",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "lint": "yarn workspaces foreach -A run lint",
+    "format": "yarn workspaces foreach -A run format",
+    "format:check": "yarn workspaces foreach -A run format:check"
```

Comment:
ok, but then we need to add this lint:check imo

Reply:
done

Reply:
add a lint:check also, for the docs we won't use NextJS so we can check the lint without fixing (and have CI fail on ESLint warnings not just errors like Next)

Reply:
and add lint:check in front with the same value as lint currently, and change CI to use lint:check

or different CI jobs for each workspace as suggested previously, as you want